data_collection/gazette/spiders/rs_lajeado.py

```diff
@@ -0,0 +1,63 @@
+from datetime import date
+from urllib.parse import urlencode
+
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RsLajeadoSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4311403"
+    BASE_URL = "https://lajeado.rs.gov.br"
+
+    name = "rs_lajeado"
+    start_date = date(2016, 4, 5)
+    params = {
+        "titulo": "Diário Oficial",
+        "template": "conteudo",
+        "codigoCategoria": 1012,
+    }
+
+    def start_requests(self):
+        params = urlencode(self.params)
+        yield Request(self.BASE_URL + "/?" + params)
+
+    def parse(self, response):
+        pdfs = response.css("ul.anexos li a")
+        for item in pdfs:
+            filename = item.css("::text").get()
```

Comment:
**issue**: We are able to get `edition_number` from this text in all links. For example: `22/11/17_edição_0396 ` we should return `0396` as `edition_number`. This is an important information to be collected.